### PR TITLE
Cloud Gateways: Chore PR

### PR DESCRIPTION
**Pull request by Guaris**
https://konghq.atlassian.net/browse/DOCU-3764

* Adds API spec
* Adds links to the CGW docs 
* Rewords and changes some terminology 
* Moves overview page from introduction section to gateway-manager section.
* Remove soc compliance
* Updated links to CGW overview (was a 404 in the nav, some pages pointed to the old link)
* Updated the data plane node use case table to include self-managed

### CODE REVIEW

File: app/konnect/dedicated-cloud-gateways.md [hunk 1]
```diff
@@ -41,4 +41,9 @@ Control planes in {{site.konnect_short_name}} **cannot** contain both Dedicated
 
 ## More information
 
-[Link out to other CGW sections and the API spec, currently we don't have those sections drafted, so I don't have anything to link to]
+[Networking and Peering information](/konnect/network-resiliency/#how-does-network-peering-work-with-dedicated-cloud-gateway-nodes)
+[Supported regions](/konnect/geo/#dedicated-cloud-gateways)
+[Transit Gateways](/konnect/gateway-manager/data-plane-nodes/transit-gateways/)
+[How to upgrade data planes](/konnect/gateway-manager/data-plane-nodes/upgrade/)
+[Custom Domains](/konnect/reference/custom-dns/)
+[Cloud Gateways API](/konnect/api/cloud-gateways/latest/)
```
Comment (lena-larionova) [inline app/konnect/dedicated-cloud-gateways.md:49]:
```suggestion
* [Networking and Peering information](/konnect/network-resiliency/#how-does-network-peering-work-with-dedicated-cloud-gateway-nodes)
* [Supported regions](/konnect/geo/#dedicated-cloud-gateways)
* [Transit Gateways](/konnect/gateway-manager/data-plane-nodes/transit-gateways/)
* [How to upgrade data planes](/konnect/gateway-manager/data-plane-nodes/upgrade/)
* [Custom Domains](/konnect/reference/custom-dns/)
* [Cloud Gateways API](/konnect/api/cloud-gateways/latest/)
```

File: app/konnect/gateway-manager/control-plane-groups/index.md [hunk 2]
```diff
@@ -203,3 +203,4 @@ Conflict detection in a control plane group happens only after you have added a
 * [Set up and manage control planes](/konnect/gateway-manager/control-plane-groups/how-to/)
 * [Migrate configuration into a control plane group](/konnect/gateway-manager/control-plane-groups/migrate/)
 * [Conflicts in control planes](/konnect/gateway-manager/control-plane-groups/conflicts/)
+* [Troubleshooting data planes](/konnect/gatway-manager/troubleshooting)
```
Comment (lena-larionova) [inline app/konnect/gateway-manager/control-plane-groups/index.md:206]:
```suggestion
* [Troubleshooting data planes](/konnect/gatway-manager/troubleshooting/)
```

File: app/konnect/gateway-manager/dedicated-cloud-gateways.md [hunk 3]
```diff
@@ -41,4 +42,9 @@ Control planes in {{site.konnect_short_name}} **cannot** contain both Dedicated
 
 ## More information
 
-[Link out to other CGW sections and the API spec, currently we don't have those sections drafted, so I don't have anything to link to]
+[Networking and Peering information](/konnect/network-resiliency/#how-does-network-peering-work-with-dedicated-cloud-gateway-nodes)
+[Supported regions](/konnect/geo/#dedicated-cloud-gateways)
+[Transit Gateways](/konnect/gateway-manager/data-plane-nodes/transit-gateways/)
+[How to upgrade data planes](/konnect/gateway-manager/data-plane-nodes/upgrade/)
+[Custom Domains](/konnect/reference/custom-dns/)
+[Cloud Gateways API](/konnect/api/cloud-gateways/latest/)
```
Comment (Guaris) [inline app/konnect/gateway-manager/dedicated-cloud-gateways.md:50]:
```suggestion
* [Networking and Peering information](/konnect/network-resiliency/#how-does-network-peering-work-with-dedicated-cloud-gateway-nodes)
* [Supported regions](/konnect/geo/#dedicated-cloud-gateways)
* [Transit Gateways](/konnect/gateway-manager/data-plane-nodes/transit-gateways/)
* [How to upgrade data planes](/konnect/gateway-manager/data-plane-nodes/upgrade/)
* [Custom Domains](/konnect/reference/custom-dns/)
* [Cloud Gateways API](/konnect/api/cloud-gateways/latest/)
```

File: app/konnect/index.md [hunk 4]
```diff
@@ -74,11 +74,9 @@ across the organization’s applications.
 
 ### Gateway Manager
 
-[Gateway Manager](/konnect/gateway-manager/) empowers your teams to securely
-collaborate and manage their own set of data plane nodes and services without
-the risk of impacting other teams and projects. Gateway Manager instantly
+[Gateway Manager](/konnect/gateway-manager/) instantly
 provisions a hosted {{site.base_gateway}} control plane and supports securely
-attaching data plane nodes.
+attaching self-hosed or [managed](/konnect/gateway-manager/dedicated-cloud-gateways/) data planes.
```
Comment (Guaris) [inline app/konnect/index.md:79]:
```suggestion
attaching self-hosted or [managed](/konnect/gateway-manager/dedicated-cloud-gateways/) data planes.
```

File: app/konnect/geo.md [hunk 5]
```diff
@@ -23,13 +23,19 @@ Geographic regions allow you to also operate {{site.konnect_saas}} in a similar
...
-* EU-west
-* EU-central
-* AP-northeast
-* AP-southeast-1
+
+* North America:
+    * Ohio (us-east-2)
+    * Oregon (us-west-2)
+* Europe:
+    * Frannkfurt (eu-central-1)
```
Comment (lena-larionova) [inline app/konnect/geo.md:31]:
```suggestion
    * Frankfurt (eu-central-1)
```

**Review by lena-larionova — APPROVED** [review]
LGTM, just one typo

**Pull request merged**